src/misc.h

```diff
@@ -81,7 +81,11 @@ struct PipeDeleter {
     void operator()(FILE* file) const {
         if (file != nullptr)
         {
+#if defined(_MSC_VER)
```

Comment:
Or maybe push the `PipeDeleter` in the conditional on Linux just below? It's used nowhere else I think.